### DISCUSSION TITLE
blog/new-chapter-begins/

### DISCUSSION THREAD

**Discussion by giscus** [harrymkt/harrymkt.github.io#4] [Comments]
# blog/new-chapter-begins/

https://harrymkt.github.io/blog/new-chapter-begins/

**Comment by harrymkt** [comment]
* Using comments.
* Used within github account.
* Markdown syntax works.

**Reply by harrymkt** [reply]
Reply work! yay!

**Comment by harrymkt** [comment]
# h1
## h2
### h3
```python
print("Hello")
```

**Comment by harrymkt** [comment]
Comment works

**Comment by patricus3** [comment]
test
